src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -300,4 +323,27 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			'width': this.getWidth()
 		});
 	}
+
+	public get editableDropdownPlaceholder(): string {
+		return this.getPropertyOrDefault<string>((props) => props.editableDropdownPlaceholder, '');
+	}
+
+	public get validationErrorMessage(): string {
```

Comment:
Did you verify that this is handled by screen readers correctly? Announced properly and in a way that lets users understand what the message is referring to?